tools/mobdbconverter.py

```diff
@@ -0,0 +1,279 @@
+#! /usr/bin/env python
+# -*- coding: utf8 -*-
+#
+# Copyright (C) 2015  Andrei Karas (4144)
+
+# This Script converts mob_db.txt to mob_db.conf format
+
+import os
+import re
+import sys
+from sets import Set
```

Comment:
The module `sets` looks like it was deprecated in python 2.6 and removed in python 3. Removing this line allows compatibility with all recent version of python (python 2.6 was released 7 years ago)